src/ansys/fluent/core/utils/fluent_version.py

```diff
@@ -20,3 +38,99 @@ def get_version_for_file_name(version: Optional[str] = None, session=None):
         version = get_version(session)
 
     return "".join(version.split(".")[0:2])
+
+
+@total_ordering
+class FluentVersion(Enum):
+    """An enumeration over supported Fluent versions.
+
+    Examples
+    --------
+    FluentVersion(None) == <FluentVersion.default: "">
+
+    FluentVersion("23.2.0") == FluentVersion.v23R2
+
+    int(FluentVersion.v23R2) == 232
+
+    str(FluentVersion.v23R2) == 'AWP_ROOT232'
+
+    FluentVersion.v23R2.value == '23.2.0'
+    """
+
+    v24R2 = "24.2.0"
+    v24R1 = "24.1.0"
+    v23R2 = "23.2.0"
+    v23R1 = "23.1.0"
+    v22R2 = "22.2.0"
+    default = ""
+
+    @classmethod
+    def _missing_(cls, version):
+        if not version:
+            return FluentVersion.default
+        if isinstance(version, (int, float, str)):
+            version = str(version)
+            if len(version) == 3:
+                version = version[:2] + "." + version[2:]
+            version += ".0"
+            for v in FluentVersion:
+                if version == v.value:
+                    return FluentVersion(version)
+            else:
+                raise RuntimeError(
```

Comment:
Should we use a higher level exception rather than a `RuntimeError`? Can use the existing `AnsysVersionNotFound`, with a custom message.